pkg/mcs/exportspec_test.go

```diff
@@ -78,25 +97,56 @@ func TestCompatibility(t *testing.T) {
 		other      *mcs.ExportSpec
 		compatible bool
 		field      string
+		mutate     func(mcs.ExportSpec) *mcs.ExportSpec
 	}{
-		"compatible: empty": {spec: &mcs.ExportSpec{}, other: &mcs.ExportSpec{}, compatible: true},
-		/*
-			"compatible: identical":        {},
-			"compatible: local properties": {},
-			"conflicting: type":            {},
-			"conflicting: affinity":        {},
-			"conflicting: port":            {},
-		*/
+		"compatible: empty": {spec: &mcs.ExportSpec{}, compatible: true,
+			mutate: func(es mcs.ExportSpec) *mcs.ExportSpec {
+				return &mcs.ExportSpec{}
+			}},
+
+		"compatible: identical": {spec: &sample, compatible: true,
+			mutate: func(es mcs.ExportSpec) *mcs.ExportSpec {
+				return &es
+			}},
+
+		"compatible: local properties": {spec: &sample, compatible: true, //#TODO check if that whay Etai means. Does it
```

Comment:
I wasn't sure if that's the local properties different that you meant,
I'm not sure if it make sense that two services will have different clusterId so I didnt changed it (does it suppose to be services from different cluster or not?).

Reply:
Local properties are not part of the service specification. 
For ExportSpecs of two clusters the ClusterID would by definition be different and they should not conflict.
However - we could (and probably should) change `IsConflictingWith()` to confirm that the namespace and name are identical as well, even though they're local properties. The exports are supposed to represent the *same* service between multiple clusters, so namespace and name ought to match.

Reply:
If we add name and namespace to the conflicting fields definition, you should change the test below and also add tests for incompatible name and/or namespace